appveyor.yml

```diff
@@ -0,0 +1,10 @@
+version: 0.1.{build}
```

Comment:
Can this file be renamed to `.appveyor.yml` (So that it is hidden on unixy platforms), or moved to a different directory? We would rather not clutter up the root of this project.